fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/FHIRTestTransactionFactory.java

```diff
@@ -14,6 +14,7 @@
  * of the {@link FHIRPersistenceTransaction} interface. Typically used
  * to support unit-tests
  */
+@Deprecated
```

Comment:
surprised this is in src/main; is anything other than test code using it?
what replaces it?

Reply:
I don't know - it isn't being used. I thought I'd deprecate it so we could discuss in this review. If you don't want it, I'll delete it.

Reply:
Yeah, I'm thinking we can just blow this one away.

Reply:
Great. Was able to delete some related classes which weren't also weren't used: FHIRUserTransactionFactory, FHIRTransactionFactory